unison-cli/src/Unison/LSP/Conversions.hs

```diff
@@ -19,8 +19,8 @@ annToInterval ann = annToRange ann <&> rangeToInterval
 uToLspPos :: Lex.Pos -> Position
 uToLspPos uPos =
   Position
-    { _line = fromIntegral $ Lex.line uPos - 1, -- 1 indexed vs 0 indexed
-      _character = fromIntegral $ Lex.column uPos - 1
+    { _line = fromIntegral $ max 0 (Lex.line uPos - 1), -- 1 indexed vs 0 indexed
+      _character = fromIntegral $ max 0 (Lex.column uPos - 1)
```

Comment:
Do you happen to know what the [existing] comment means? Is it converting between indexing schemes, or is it referring to the input or the output?

Reply:
It's converting between LSP Ranges and Unison Ranges; unison ranges are 1-indexed (e.g. the first line is line 1) whereas LSP the first line is line 0.

Reply:
Got it. Could you fix up the comment to that effect while we're here; as Haddock on `uToLspPos` or wherever you think is best. and then good to go IMO